mypy/subtypes.py

```diff
@@ -204,27 +238,24 @@ def visit_type_var(self, left: TypeVarType) -> bool:
         right = self.right
         if isinstance(right, TypeVarType) and left.id == right.id:
             return True
-        if left.values and is_subtype(UnionType.make_simplified_union(left.values), right):
+        if left.values and self._is_subtype(UnionType.make_simplified_union(left.values), right):
             return True
-        return is_subtype(left.upper_bound, self.right)
+        return self._is_subtype(left.upper_bound, self.right)
 
     def visit_callable_type(self, left: CallableType) -> bool:
         right = self.right
         if isinstance(right, CallableType):
             return is_callable_compatible(
                 left, right,
-                is_compat=is_subtype,
+                is_compat=self._is_subtype,
                 ignore_pos_arg_names=self.ignore_pos_arg_names)
         elif isinstance(right, Overloaded):
-            return all(is_subtype(left, item, self.check_type_parameter,
-                                  ignore_pos_arg_names=self.ignore_pos_arg_names)
-                       for item in right.items())
+            return all(self._is_subtype(left, item) for item in right.items())
         elif isinstance(right, Instance):
-            return is_subtype(left.fallback, right,
-                              ignore_pos_arg_names=self.ignore_pos_arg_names)
+            return is_subtype(left.fallback, right)
```

Comment:
Why are you using `is_subtype` instead of `self._is_subtype`? If this is some special case, then it deserves a comment.

Reply:
Ah whoops -- that was an oversight. Fixed.